### DISCUSSION TITLE
Release Date for Open Source Badge?

### DISCUSSION THREAD

**Discussion by suhaib-mousa** [community/community#133885] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hey, When will the open-source badge be released? I believe it will enhance our profiles.

![213864050-4f9b1f0d-03cc-4778-9d7f-df69bfc4c0a1](https://github.com/user-attachments/assets/623d9ebb-b0a4-4cd9-b7b6-555c3ea694ce)

**Comment by subhanjandas** [comment, marked as answer]
The 'Open Sourcerer' and 'Heart On Your Sleeve' achievement badges are currently not available to the public on GitHub. To stay updated on their release, I recommend keeping an eye on the official GitHub blog. When GitHub introduced the first set of achievement badges, they made an announcement on their blog, and they are likely to do the same for these new badges.

You can check for updates and announcements on the GitHub blog [here](https://github.blog/). This will provide the most accurate and official information about the mechanics and availability of these badges.

Hope this helps! Stay tuned for the official updates, and take care! 😄
